llvm/test/MC/AMDGPU/gfx11_asm_vop3_dpp16.s

```diff
@@ -4517,183 +4682,18 @@ v_fma_f16_e64_dpp v5, v1, -|v2|, -|0.5| op_sel:[0,0,1,0] row_xmask:0 row_mask:0x
 v_fma_f16_e64_dpp v255, -|v255|, -|v255|, -|src_scc| op_sel:[0,0,0,1] clamp row_xmask:15 row_mask:0x3 bank_mask:0x0 bound_ctrl:1 fi:1
 // GFX11: [0xff,0xc7,0x48,0xd6,0xfa,0xfe,0xf7,0xe3,0xff,0x6f,0x0d,0x30]
 
-v_mad_i16_e64_dpp v5, v1, v2, exec_hi op_sel:[1,1,1,1] row_ror:15 row_mask:0xf bank_mask:0xf
-// GFX11: [0x05,0x78,0x53,0xd6,0xfa,0x04,0xfe,0x01,0x01,0x2f,0x01,0xff]
```

Comment:
Looks these have been moved.

Reply:
This one is moved to line 3355. I've manually checked all op_sel and move them to a grouped location

Reply:
I believe it was discussed (and evidenced) before that reshuffling tests complicates reviews and merges, and should be done with separate patches, if can't be avoided.